ujt/callbacks/sli_refresh_callbacks.py

```diff
@@ -0,0 +1,31 @@
+import dash
+from dash.dependencies import Input, Output
+
+from .. import (
+    id_constants,
+    utils,
+)
+from ..dash_app import app
+
+
+@app.callback(
+    Output(id_constants.SIGNAL_SLI_REFRESH, "children"),
+    [
+        Input(id_constants.REFRESH_SLI_BUTTON, "n_clicks_timestamp"),
+        Input(id_constants.REFRESH_SLI_INTERVAL, "n_intervals"),
+    ],
+    prevent_initial_call=True,
+)
+def update_sli_refresh_signal(n_clicks_timestamp: int, n_intervals: int):
+    """Updates the SLI refresh signal.
```

Comment:
We could have skipped this signal creation step and directly register the interval component and the button as inputs to update_graph_elements. 

However, I chose this approach to reduce the amount of inputs to update_graph_elements and make this more extensible in the future if other features require refreshing the graph.